scripts/assemble_visualization_webpage.py

```diff
@@ -208,15 +226,60 @@ def main(
         fd.write(mod_html)
 
 
+def main():
+    """Parse command line, run default functions."""
+    import argparse
+    # parse command line
+    # create the top-level parser
+    parser = argparse.ArgumentParser(description="Generate HTML visual report from VCF variants")
+    parser.add_argument('-f','--consensus', metavar='FASTA', required=False,
+        type=str, dest='consensus_file', help="consensus sequence of this sample")
+    parser.add_argument('-c','--coverage', metavar='TSV', required=False,
+        default='variants/coverage.tsv',
+        type=str, dest='coverage_file', help="global coverage table")
+    parser.add_argument('-v','--vcf', metavar='VCF', required=False,
+        type=str, dest='vcf_file', help="VCF containing the SNPs to be visualised")
+    parser.add_argument('-g','--gff', metavar='DIR', required=False,
+        type=str, dest='gff_directory', help="directory containing GFF annotations")
+    parser.add_argument('-p','--primers', metavar='CSV', required=False,
+        type=str, dest='primers_file', help="table with primers")
+    parser.add_argument('-m','--metainfo', metavar='YAML', required=False,
+        type=str, dest='metainfo_yaml', help="metainformation for the GFF and primers")
+    parser.add_argument('-t','--template', metavar='HTML', required=False,
+        default=f'{os.path.dirname(__file__)}/visualization.html',
+        type=str, dest='html_file_in', help="HTML template used to generate visual report")
+    parser.add_argument('-o','--output', metavar='HTML', required=False,
+        type=str, dest='html_file_out', help="produced HTML report")
+    parser.add_argument('-w','--wildcards', metavar='SAMPLE/DATE', required=False,
+        type=str, dest='wildcards_dataset', help="sample's two-level directory hierarchy prefix")
+    parser.add_argument('-r','--reference', metavar='FASTA', required=False,
+        default='variants/cohort_consensus.fasta',
+        type=str, dest='reference_file', help="reference against which SNVs were called (e.g.: cohort's consensus)")
+
+    args = parser.parse_args()
+
+    # defaults which can be guess from one another
```

Comment:
I would move all the file checks for the required files here. (eg instead of checking in the beginning of each function, eg the check in convert_vcf)

I think it's nicer to do this here and assume file existence in every other function, since these functions are only used internally here and are not library functions.